### ISSUE TITLE
"allman" option of `brace-style` rule does not perform as documented

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3486]
According to the documentation, these should cause an error:

```js
/*eslint brace-style: [2, "allman"]*/

if (foo)
{
  bar(); }

try
{
  somethingRisky();
} catch(e)
{
  handleError();
}
```

Using current master, I get no errors.

**Comment by BYK** [comment]
My, bad. Looking into it.

**Comment by BYK** [comment]
Welp, seems like we have a bug in `stroustrup` style too for closing braces.

**Comment by BYK** [comment]
Even more errors: almost nothing warns if you have the brace on the same line with the statement. :(
